src/components/community-link/community-link.js

```diff
@@ -0,0 +1,81 @@
+import '@brightspace-ui/core/components/link/link.js';
+import '@brightspace-ui/core/components/button/button-icon.js';
+import { html, LitElement } from 'lit';
+import { getDocumentLocaleSettings } from '@brightspace-ui/intl/lib/common.js';
+
+class CommunityLink extends LitElement {
+
+	static get properties() {
+		return {
+			text: { type: String },
+			type: { type: String },
+			articleMap: { attribute: 'article-map', type: Object }
+		};
+	}
+
+	constructor() {
+		super();
+		this._localeSettings = getDocumentLocaleSettings();
+		this._handleUpdate = () => this.requestUpdate();
+	}
+
+	get link() {
+		const { language } = getDocumentLocaleSettings();
+		const { articleCode, langCode } = this._getArticleCode(language);
+
+		return `https://community.d2l.com/brightspace${langCode === 'en' ? '' : `-${langCode}`}/kb/articles/${articleCode}`;
+	}
+
+	connectedCallback() {
+		super.connectedCallback();
+		this._localeSettings.addChangeListener(this._handleUpdate);
+	}
+
+	disconnectedCallback() {
+		super.disconnectedCallback();
+		this._localeSettings.removeChangeListener(this._handleUpdate);
+	}
+
+	render() {
+		return this.type === 'button'
+			? html`<d2l-button-subtle
```

Comment:
So unfortunately by using subtle buttons (which render native `<button>`), the link semantics are gone. The ideal scenario here would be a `<a>` that's styled to look like a button, but like I said we still try to avoid this whenever possible since the user loses all the link niceties like right-clicking.

But this _is_ a labs component so I don't want to get super strict in here...

Reply:
Yeah it's not opimal. I copied this pattern from the insights dashboards where it looks like this

![community link](https://github.com/BrightspaceUI/labs/assets/14355188/eea2e82f-7bf7-4e7c-a488-e4ae96646083)

Reply:
It's a subtle button because the overflow panel shows the button witht he icon (sometimes lol)